.babelrc

```diff
@@ -1 +1,4 @@
-{ "presets": ["es2015"] }
+{
+  "presets": ["es2015"],
+  "plugins": ["add-module-exports"]
```

Comment:
This is not the urgent work so I'll close the PR and welcome anyone to make a full patch